src/config.rs

```diff
@@ -848,8 +854,10 @@ impl Config {
         }
 
         // log_level
-        match (matches.occurrences_of("verbose"),
-                                            matches.occurrences_of("quiet")) {
+        match (
```

Comment:
I don't think this is new in this PR per se, but this code doesn't make sense (anymore) with the `.conflicts_with` on the `verbose`/`quiet` args. Running the binary will not get past the arg parsing anyway.

Reply:
Hm, yeah. But the code is cute ;)